### PR TITLE
fix: subscriptions\vs-rstudio.md

### PR DESCRIPTION
**Pull request by nschonni**
- MD012/no-multiple-blanks
- Trim trailing space
- Space around headers

### CODE REVIEW

**Comment by PRMerger8** [comment]
@nschonni : Thanks for your contribution! The author, @evanwindom, has been notified to review your proposed change.

File: subscriptions/vs-rstudio.md [hunk 1]
```diff
@@ -100,27 +97,30 @@ RStudio is available in open source and commercial editions and runs on the desk
...
 -  For assistance with sales, subscriptions, accounts and billing for Visual Studio Subscriptions, contact Visual Studio [Subscriptions Support](https://visualstudio.microsoft.com/subscriptions/support/).
 -  Have a question about Visual Studio IDE, Azure DevOps Services or other Visual Studio products or services?  Visit [Visual Studio Support](https://visualstudio.microsoft.com/support/).
 
-
-
 ## Next steps
-The RStudio benefit is available for a very limited time.  Remember to claim your code and create your RStudio account before July 1, 2019.  
\ No newline at end of file
+
+The RStudio benefit is available for a very limited time.  Remember to claim your code and create your RStudio account before July 1, 2019.
```
Comment (nschonni) [inline subscriptions/vs-rstudio.md]: Maybe this should be at the top with an `[!IMPORTANT]` or `[!NOTE]`

Comment (evanwindom) [reply subscriptions/vs-rstudio.md]: @nschonni - Thanks for the feedback.  The note is already there, however.

Comment (nschonni) [reply subscriptions/vs-rstudio.md]: Gotcha, I see it up on L19 now

**Comment by PRMerger6** [comment]
@Jak-MS : Thanks for your contribution! The author, @evanwindom, has been notified to review your proposed change.

**Pull request merged**